Cargo.toml

```diff
@@ -138,7 +138,7 @@ rand = "0.8"
 rstest = "0.11.0"
 rupnp = { version = "1.1.0", default-features = false }
 rustc-hash = "1"
-serde = { version = "1.0", features = ["derive"] }
+serde = { version = "=1.0.197", features = ["derive"] }
```

Comment:
Why?

Reply:
`cargo` fails to select `serde` versions.

Reply:
Why not just call `cargo update -p serde`? Or putting here `>=1.0.197` at least? I'm worried `=1.0.197` can cause additional problems next time some other dependency updates and requires newer version

Reply:
Yeah, good idea, I just blindly followed cargo's suggestion :smile:  fixed.